EventKit/eventkit-cloud
# Fixes missing supports for create_aprx script when using templates.

## Description of Improvement

Fixes a bug in the create_aprx script when using the layer templates.

## How to test

Create an export that would use the layer templates (i.e. elevation) and ensure that the create_aprx script completes as expected.

## Quality Assurance 

The following items have been updated:
 
## Screenshots



:smile:
